tools/dev/import-dev-keys.sh

```diff
@@ -13,47 +11,3 @@ for key in $KEYBASE_KEY_URLS; do
   echo Importing $key...
   \curl -sSL $key | gpg --quiet --no-auto-check-trustdb --import -
 done
-
-# Exceptions -- keys that do show up in the logs, but aren't (yet) in Keybase:
-# This should cover every key since May of 2014.
-
-# Currently, one lone missing key:
-#
-# gpg: Signature made Mon 16 Feb 2015 02:09:53 PM CST using RSA key ID D5D50A02
-# gpg: Can't check signature: public key not found
-# 14da69c - Land #4757, adds RC for auto payload gen (3 months ago) <kernelsmith@github> []
-#
-# https://github.com/rapid7/metasploit-framework/commit/14da69c is
-# harmless, though. It's only an RC script, not run by default, and it
-# automates setting up a payload handler.
-
-
-echo Processing exceptions...
-
-UBUNTU_KEYIDS="
-Brandont       0xA3EE1B07
-Ccatalan       0xC3953653
-Farias         0x01DF79A1
-Firefart       0x66BC32C7
-HDM            0xFA604913
-Jvennix        0x3E85A2B0
-Kernelsmith    0x3D609E33
-Lsanchez       0xFB80E8DD
-OJ             0x1FAA5749
-Sgonzalez      0xCA93BCE5
-Shuckins       0x8C03C944
-TheLightCosine 0x3A913DB2
-Wvu            0xC1629024
-"
-
-UBUNTU_KEY_URL_BASE="https://keyserver.ubuntu.com/pks/lookup?op=get&search="
```

Comment:
Almost certain this will break things if we remove this. Do we have a good reason for removing this specifically vs the entire script? This seems to still be correctly retrieving people's keys so I don't think this code per say is wrong.

Reply:
I'd rather we remove the entire script, but to answer your question, most of these people haven't been devs in several years.

Reply:
(And look when my `0xC1629024` key expired.)

Reply:
In particular, keys that are a liability should be removed. I don't control that key anymore, and I wasn't able to revoke it.

Reply:
As I understand it, the authoritative list is https://docs.metasploit.com/docs/development/maintainers/committer-keys.html (generated).